components/case sudies/CaseStudiesCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Image from 'next/image';
+
+import { CaseStudiesCardProps } from '@/types';
+
+const CaseStudiesCard: React.FC<CaseStudiesCardProps> = ({
+  altText,
+  projectName,
+  heading,
+  subHeading,
+  image,
+  _id,
+}) => {
+  const projectBackgrounds: Record<string, string> = {
```

Comment:
Maybe `projectColorBackgrounds` is a better name as more descriptive.

Reply:
Resolved